### ISSUE TITLE
Universal Search for Events

### ISSUE THREAD

**Issue by syloktools** [MISP/MISP#2959]
Feature Request

Add the ability to filter the Events view in the GUI. You should be able to filter on aspects such as Published, Organization, User, Events that contain certain tags, etc. Also you should be able to set default so that you do not have to click Org Events every time you get to the page. It should also remember where you are in search or list after you click on something or open an event. Right now it brings you all the way back to the beginning.

**Comment by zeroq** [comment]
<img width="1183" alt="screen shot 2018-07-18 at 13 07 20" src="https://user-images.githubusercontent.com/2749718/42877943-9933f3e4-8a8b-11e8-9c56-20d79b2a0983.png">

Started to play around with QueryBuilder and Datatables ... lets see how far I get with my basic cakephp skills ;)

**Comment by iglocska** [comment]
Woaah! That looks great!

**Comment by SteveClement** [comment]
@zeroq which branch are you working on?
You should fork and push to your fork :)

But looks really good so far!

**Comment by zeroq** [comment]
Good point ;) Yes, I will fork and work on the fork, currently it's just a local copy
